index.js

```diff
@@ -1 +1,15 @@
-module.exports = 'a[href], area[href], input:not([disabled]), select:not([disabled]), textarea:not([disabled]), button:not([disabled]), iframe, object, embed, [tabindex="0"], [contenteditable], audio[controls], video[controls], summary'
+module.exports =
+  'a[href],' +
+  'area[href],' +
+  'input:not([disabled]),' +
+  'select:not([disabled]),' +
+  'textarea:not([disabled]),' +
+  'button:not([disabled]),' +
+  'iframe,' +
+  'object,' +
+  'embed,' +
+  '[tabindex]:not([tabindex="-1"]),' +
```

Comment:
This will treat `<div tabindex="-2">` as a focusable element. It'd also allow for `<div tabindex="hello">` to be returned. While we probably don't need to worry about `tabindex="hello"`, we should filter out all negative values, not just -1.

Reply:
Thank you for pointing out those edge cases!

I ended up with function which will generate `[tabindex^="0"],[tabindex^="1"],[tabindex^="2"],[tabindex^="3"],[tabindex^="4"],[tabindex^="5"],[tabindex^="6"],[tabindex^="7"],[tabindex^="8"],[tabindex^="9"]`.

Another edge case is `<div tabindex>` which shouldn't be focusable as well.